### PR TITLE
Map more rules into Anssi policy

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Maps rules into ANSSI requirements
- Select mapped rules in profile corresponding to hardening level

### CODE REVIEW

**Comment by matejak** [comment]
It looks like that we have double-selection of rules `sysctl_fs_suid_dumpable` and `sysctl_kernel_randomize_va_space`.

File: rhel7/profiles/anssi_nt28_intermediary.profile [hunk 1]
```diff
@@ -22,6 +35,16 @@ selections:
     - sysctl_fs_suid_dumpable
     - sysctl_kernel_randomize_va_space
```
Comment (matejak) [inline rhel7/profiles/anssi_nt28_intermediary.profile]: first occurrence of both

File: rhel7/profiles/anssi_nt28_intermediary.profile [hunk 2]
```diff
@@ -124,3 +147,49 @@ selections:
...
+    - sysctl_fs_suid_dumpable
+
+    # Prohibit links to find links to files
+    # the current user is not the owner
+    # Can prevent some programs from working properly
+    - sysctl_fs_protected_symlinks
+    - sysctl_fs_protected_hardlinks
+
+    # Activation of the ASLR
+    - sysctl_kernel_randomize_va_space
```
Comment (matejak) [inline rhel7/profiles/anssi_nt28_intermediary.profile]: second occurrence

File: rhel7/profiles/anssi_nt28_intermediary.profile [hunk 3]
```diff
@@ -124,3 +147,49 @@ selections:
...
     # net.ipv6.conf.default.max_addresses = 1
+
+    # ==============================================
+    #  R23 - Setting system sysctl
+    # Here is a list of recommended system sysctl (in the format /etc/sysctl.conf):
+    # Disabling SysReq
+    # kernel. sysrq = 0
+
+    # No core dump of executable setuid
+    - sysctl_fs_suid_dumpable
```
Comment (matejak) [inline rhel7/profiles/anssi_nt28_intermediary.profile]: second occurrence

**Comment by yuumasato** [comment]
@matejak Thanks for review, and nice catch.
The duplicated rules have been removed.

**Comment by matejak** [comment]
Thanks for improving the ANSSI profile set!

**Pull request merged**